stubs/GraphQL/Parser.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace GraphQL;
+
+class Parser
+{
+    /**
+     * @param string $input
+     *
+     * @return array
+     */
+    public function parse(string $input): array {
```

Comment:
This doesn't works for versions before 7.0 right? Can you remove the type hint and the return type?